Sample-01/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "login-demo",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e",
+    "server": "node server.js",
+    "dev": "npm-run-all --parallel start server",
+    "prod": "ng build --prod --aot && NODE_ENV=production node server",
+    "postbuild": "npm run copy:server",
+    "copy:server": "cp server.js auth_config.json dist/login-demo"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "~10.0.8",
+    "@angular/common": "~10.0.8",
+    "@angular/compiler": "~10.0.8",
+    "@angular/core": "~10.0.8",
+    "@angular/forms": "~10.0.8",
+    "@angular/localize": "^10.0.8",
+    "@angular/platform-browser": "~10.0.8",
+    "@angular/platform-browser-dynamic": "~10.0.8",
+    "@angular/router": "~10.0.8",
+    "@auth0/auth0-angular": "file:../../../../auth0-auth0-angular-0.1.0.tgz",
+    "@fortawesome/angular-fontawesome": "^0.7.0",
+    "@fortawesome/fontawesome-svg-core": "^1.2.19",
+    "@fortawesome/free-solid-svg-icons": "^5.9.0",
+    "express": "^4.17.1",
+    "express-jwt": "^5.3.1",
```

Comment:
🚨 this should be 6+. I suggest you run an `npm audit fix` and then run `npm audit` to see if there were any "breaking change" warnings left over.

Reply:
Yep, thanks for the reminder. I had planned to upgrade all dependencies once I'd done the core set for Angular. I'll push an update now.